javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -191,6 +191,7 @@ object Character {
     (c & HighSurrogateMask) == HighSurrogateID
   @inline def isLowSurrogate(c: scala.Char): scala.Boolean =
     (c & LowSurrogateMask) == LowSurrogateID
+  @inline def isSurrogate(c: scala.Char): scala.Boolean = isHighSurrogate(c) || isLowSurrogate(c)
```

Comment:
Avoid lines longer than 80 characters if it does not hurt readability. Here you can easily put the body of the method on the next line, like the other ones.